cpp/ql/test/experimental/query-tests/Security/CWE/CWE-788/semmle/tests/AccessOfMemoryLocationAfterEndOfBufferUsingStrlen.expected

```diff
@@ -0,0 +1,9 @@
+| test.c:13:3:13:24 | ... = ... | use a different method to calculate the length. |
+| test.c:14:3:14:40 | ... = ... | use a different method to calculate the length. |
+| test.c:15:3:15:40 | ... = ... | use a different method to calculate the length. |
+| test.c:16:3:16:44 | ... = ... | use a different method to calculate the length. |
+| test.c:17:3:17:44 | ... = ... | use a different method to calculate the length. |
+| test.c:18:3:18:48 | ... = ... | use a different method to calculate the length. |
+| test.c:19:3:19:48 | ... = ... | use a different method to calculate the length. |
+| test.c:20:3:20:50 | ... = ... | use a different method to calculate the length. |
+| test.c:21:3:21:50 | ... = ... | use a different method to calculate the length. |
```

Comment:
The PR checks now fail because the alert message has been updated:

```
2021-01-27T07:42:56.5513542Z Executing tests in /home/runner/work/semmle-code/semmle-code/ql/cpp/ql/test/experimental/query-tests/Security/CWE/CWE-788/semmle/tests.
2021-01-27T07:42:57.1086333Z --- expected
2021-01-27T07:42:57.1086936Z +++ actual
2021-01-27T07:42:57.1087532Z @@ -1,9 +1,9 @@
2021-01-27T07:42:57.1088473Z -| test.c:13:3:13:24 | ... = ... | use a different method to calculate the length. |
2021-01-27T07:42:57.1089817Z -| test.c:14:3:14:40 | ... = ... | use a different method to calculate the length. |
2021-01-27T07:42:57.1090981Z -| test.c:15:3:15:40 | ... = ... | use a different method to calculate the length. |
2021-01-27T07:42:57.1092634Z -| test.c:16:3:16:44 | ... = ... | use a different method to calculate the length. |
2021-01-27T07:42:57.1093802Z -| test.c:17:3:17:44 | ... = ... | use a different method to calculate the length. |
2021-01-27T07:42:57.1095137Z -| test.c:18:3:18:48 | ... = ... | use a different method to calculate the length. |
2021-01-27T07:42:57.1096304Z -| test.c:19:3:19:48 | ... = ... | use a different method to calculate the length. |
2021-01-27T07:42:57.1097411Z -| test.c:20:3:20:50 | ... = ... | use a different method to calculate the length. |
2021-01-27T07:42:57.1098498Z -| test.c:21:3:21:50 | ... = ... | use a different method to calculate the length. |
2021-01-27T07:42:57.1103229Z +| test.c:13:3:13:24 | ... = ... | potential unsafe or redundant assignment. |
2021-01-27T07:42:57.1104177Z +| test.c:14:3:14:40 | ... = ... | potential unsafe or redundant assignment. |
2021-01-27T07:42:57.1105043Z +| test.c:15:3:15:40 | ... = ... | potential unsafe or redundant assignment. |
2021-01-27T07:42:57.1106031Z +| test.c:16:3:16:44 | ... = ... | potential unsafe or redundant assignment. |
2021-01-27T07:42:57.1106887Z +| test.c:17:3:17:44 | ... = ... | potential unsafe or redundant assignment. |
2021-01-27T07:42:57.1107741Z +| test.c:18:3:18:48 | ... = ... | potential unsafe or redundant assignment. |
2021-01-27T07:42:57.1108584Z +| test.c:19:3:19:48 | ... = ... | potential unsafe or redundant assignment. |
2021-01-27T07:42:57.1109435Z +| test.c:20:3:20:50 | ... = ... | potential unsafe or redundant assignment. |
2021-01-27T07:42:57.1110402Z +| test.c:21:3:21:50 | ... = ... | potential unsafe or redundant assignment. |
```

Please update this file.